core/src/main/java/io/lionweb/lioncore/java/serialization/data/UsedLanguage.java

```diff
@@ -25,6 +25,13 @@ public static UsedLanguage fromLanguage(@Nonnull Language language) {
     return new UsedLanguage(language.getKey(), language.getVersion());
   }
 
+  public static UsedLanguage fromMetaPointer(@Nonnull MetaPointer metaPointer) {
+    Objects.requireNonNull(metaPointer, "metaPointer parameter should not be null");
+    Objects.requireNonNull(metaPointer.getLanguage(), "metaPointer language should not be null");
```

Comment:
Should we have a method to validate a metaPointer and have the null checks there?

Reply:
I wrote this symmetric to `fromLanguage()`, didn't think too much about it.
In theory, we could work with a MetaPointer that misses its `key`.
I don't have a strong opinion, what do you prefer?